ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -728,30 +722,17 @@
     "ocs-storagecluster-rbdplugin-snapclass.yaml",
 ]
 
-# TODO: Remove monitoring_registry_pvc_list once issue #3465 is fixed
-# https://github.com/red-hat-storage/ocs-ci/issues/3465
-monitoring_registry_pvc_list = [
-    "my-alertmanager-claim-alertmanager-main-0.yaml",
-    "my-alertmanager-claim-alertmanager-main-1.yaml",
-    "my-alertmanager-claim-alertmanager-main-2.yaml",
-    "my-prometheus-claim-prometheus-k8s-0.yaml",
-    "my-prometheus-claim-prometheus-k8s-1.yaml",
-    "registry-cephfs-rwx-pvc.yaml",
-]
-GATHER_COMMANDS_OTHERS_EXTERNAL_EXCLUDE.extend(monitoring_registry_pvc_list)
+if config.DEPLOYMENT["external_mode"]:
+    GATHER_COMMANDS_OTHERS = GATHER_COMMANDS_OTHERS + [
```

Comment:
you can change the list name `GATHER_COMMANDS_OTHERS`  to `GATHER_COMMANDS_OTHERS_EXTERNAL`.
and do not use conditions (if)

Reply:
done.